lib/models/StoreStoryModel.ts

```diff
@@ -1,35 +1,41 @@
-import mongoose from 'mongoose'
+import mongoose, { Document } from 'mongoose'
 
-const StoreStorySchema = new mongoose.Schema(
-	{
-		storeImage: {
-			type: String,
-			required: [true, 'Every store must have an image']
-		},
-		storeDetails: {
-			type: String,
-			required: [true, 'Every store must have a story']
-		},
-		ownerImage: {
-			type: String,
-			required: [true, 'Every owner must have an image']
-		},
-		ownerDetails: {
-			type: String,
-			required: [true, 'Every owner must have a story']
-		},
-		storeId: {
-			type: mongoose.Schema.Types.ObjectId,
-			required: true,
-			ref: 'Store'
-		}
-	},
-	{
-		timestamps: true
-	}
+export interface StoreStory extends Document {
```

Comment:
This was so I could use the typescript models in my components instead of using "any"